toolbox/library/src/Jacobian.cpp

```diff
@@ -264,8 +263,8 @@ bool Jacobian::output(const BlockInformation* blockInfo)
     Map<MatrixXdiDynTree> jacobianRowMajor = toEigen(pImpl->jacobian);
     Map<MatrixXdSimulink> jacobianColMajor(
         output->getBuffer<double>(),
-        blockInfo->getOutputPortMatrixSize(OutputIndex::Jacobian).first,
-        blockInfo->getOutputPortMatrixSize(OutputIndex::Jacobian).second);
+        blockInfo->getOutputPortMatrixSize(OutputIndex::Jacobian).rows,
+        blockInfo->getOutputPortMatrixSize(OutputIndex::Jacobian).cols);
```

Comment:
👍

Reply:
This was really necessary! I regretted using `pairs` and `tuples` in this case.